### ISSUE TITLE
account update with --user is throwing schema validation failure

### ISSUE THREAD

**Issue by anandhu-karattu** [noobaa/noobaa-core#8072]
### Environment info
- NooBaa Version: **noobaa-core-5.15.3-20240514.el9.x86_64**
- Platform: **Standalone Noobaa**

### Actual behavior
If the account is created with uid and gid, we provide an option to update the username(local user) later on with account update command.
This option is throwing invalid schema error during account update.

### Expected behavior
Account should get updated with username

### Steps to reproduce
1. Create an account with uid and gid and list it.
```
[root@3node-x-worker1 ~]# noobaa-cli account list --name account1000 --wide 2>/dev/null
{
  "response": {
    "code": "AccountList",
    "reply": [
      {
        "_id": "664efb63ba9a7161464bae61",
        "name": "account1000",
        "email": "account1000",
        "creation_date": "2024-05-23T08:16:35.699Z",
        "nsfs_account_config": {
          "uid": 1000,
          "gid": 1000,
          "new_buckets_path": "/mnt/fs1/account1000",
          "fs_backend": "GPFS"
        },
        "allow_bucket_creation": true,
        "master_key_id": "664e107282dba0d07d51b455"
      }
    ]
  }
}
```

Make sure uid and gid is proper for the user on all the protocol nodes where noobaa service is running.
```
[root@3node-x-worker1 ~]# id anandhu1000
uid=1000(anandhu1000) gid=1000(testgrp1000) groups=1000(testgrp1000)
```
2. **Now try to update the user name : FAILED**

```
[root@3node-x-worker1 ~]# noobaa-cli account update --name account1000 --user anandhu1000 2>/dev/null
{
  "error": {
    "code": "InvalidSchema",
    "message": "Schema invalid, please use required properties",
    "detail": "Error: must NOT have additional properties | {\"additionalProperty\":\"distinguished_name\"} | \"/nsfs_account_config\"\n    at Object.validate_account_schema (/usr/local/noobaa-core/src/manage_nsfs/nsfs_schema_utils.js:50:15)\n    at update_account (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:482:27)\n    at async manage_account_operations (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:585:9)\n    at async account_management (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:337:5)\n    at async main (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:112:13)"
  }
}
```

**Comment by romayalon** [comment]
@shirady your [PR](https://github.com/noobaa/noobaa-core/pull/8058) should fix this one, right?

**Comment by shirady** [comment]
@romayalon as far as I understand it, yes. I'll add this issue link.

In the mentioned scenario above the account was created by passing `--uid` and `--gid` flags and then when trying to update it with the `--user` flag (instead of the uid and gid that we had) it currently throws an error (after the fix will be merged it should be updated).

**Comment by shirady** [comment]
The fix is in PR #8058
